src/azure/cli/_config.py

```diff
@@ -0,0 +1,55 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+import os
+from six.moves import configparser
+
+GLOBAL_CONFIG_PATH = os.path.expanduser(os.path.join('~', '.azure', 'az_config'))
+ACTIVE_ENV_CONFIG_PATH = os.path.expanduser(os.path.join('~', '.azure', 'env_config', 'default'))
+ENV_VAR_PREFIX = 'AZURE_CLI_'
+
+_UNSET = object()
+_ENV_VAR_FORMAT = ENV_VAR_PREFIX+'{section}_{option}'
+
+class AzConfig(object):
+    _BOOLEAN_STATES = {'1': True, 'yes': True, 'true': True, 'on': True,
```

Comment:
I followed the values used by the Python ConfigParser.
https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.getboolean
https://github.com/python-git/python/blob/master/Lib/ConfigParser.py#L345

We can choose to support y, n if we want but it'd be a customization.